ocs_ci/ocs/machine.py

```diff
@@ -120,3 +120,28 @@ def delete_machine_and_check_state_of_new_spinned_machine(machine_name):
         log.info(f"{new_machine.name} is in {state} state")
         return state == constants.STATUS_RUNNING.islower()
     return False
+
+
+def get_machinesets():
```

Comment:
Make sure there are functions to cover the node additions in VMware environment also.